pkg/operator/ceph/file/controller.go

```diff
@@ -0,0 +1,277 @@
+/*
```

Comment:
this file just moved, right? I wonder why it shows up as all new even though there is a lot of migration code in it

Reply:
it moved and also got a fair amount of new code to watch legacy CRD types and to handle migrations.  that code may actually be over 50% of the file, so maybe that's why it's showing up as new?